jest.config.js

```diff
@@ -93,7 +93,8 @@ module.exports = {
         : acc;
     }, {}),
     '.+\\.(css|styl|less|sass|scss)$':
-      '<rootDir>/node_modules/jest-css-modules-transform'
+      '<rootDir>/node_modules/jest-css-modules-transform',
+    d3: '<rootDir>/node_modules/d3/dist/d3.min.js'
```

Comment:
why was it needed to specify d3?

Reply:
`chart/utils` imports d3 and jest has problems importing packages without `exports.require` option declared (https://github.com/facebook/jest/issues/12036#issuecomment-1039479569)

Reply:
gotcha! 👍